source/manual/govuk-notify.html.md

```diff
@@ -24,16 +24,17 @@ for each environment):
 
   This service is used by Email Alert API only. It's used to send public-facing
   email updates about pieces of content on GOV.UK. It's our biggest sender of
-  emails and averages around 4 million emails per day.
+  emails generating millions of emails per day.
 
   The [Email Alert API Product dashboard] shows usage over time.
 
-  **Note:** currently (as of 30/06/2020) GOV.UK Notify has a maximum rate limit
-  of 350 requests per second meaning we should adhere to this limit when using
-  Notify to deliver emails for our applications. This is set in email-alert-api
-  and can be found in the [DeliveryRequestWorker][DeliveryRequestWorker]. If you
-  wish to double check this figure you could ask in their slack channel
-  #govuk-notify.
+  **Note:** currently (as of 04/08/2020) GOV.UK Notify has a maximum rate limit
+  of 350 requests per second and a daily limit of 30 million emails per day
+  (Notify's daily allowance with Amazon SES) meaning we should adhere to these
+  limits when using Notify to deliver emails for our applications. The rate
+  limit of 350 requests per second is set in email-alert-api and can be found in the
```

Comment:
```suggestion
  limit of 350 requests per second is set in email-alert-api and can be found in the
```
```suggestion
  limit of 350 requests per second, for sending emails to Notify, is set in email-alert-api and can be found in the
```

🤣

Reply:
Ah ha, we aren't actually sending emails to Notify, only mere requests... 

I'm merging and running away from this PR! 😆